server/src/main/java/com/objectcomputing/checkins/security/ImpersonationController.java

```diff
@@ -0,0 +1,151 @@
+package com.objectcomputing.checkins.security;
+
+import com.objectcomputing.checkins.Environments;
+import com.objectcomputing.checkins.services.memberprofile.MemberProfile;
+import com.objectcomputing.checkins.services.memberprofile.currentuser.CurrentUserServices;
+import com.objectcomputing.checkins.services.permissions.Permission;
+import com.objectcomputing.checkins.services.permissions.RequiredPermission;
+import io.micronaut.http.MutableHttpResponse;
+import io.micronaut.http.cookie.SameSite;
+import io.micronaut.http.cookie.Cookie;
+import io.micronaut.http.netty.cookies.NettyCookie;
+import io.micronaut.security.utils.SecurityService;
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.context.event.ApplicationEventPublisher;
+import io.micronaut.http.HttpRequest;
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.HttpStatus;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.Consumes;
+import io.micronaut.http.annotation.Produces;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Get;
+import io.micronaut.http.annotation.Post;
+import io.micronaut.scheduling.TaskExecutors;
+import io.micronaut.scheduling.annotation.ExecuteOn;
+import io.micronaut.security.annotation.Secured;
+import io.micronaut.security.authentication.Authentication;
+import io.micronaut.security.authentication.AuthenticationResponse;
+import io.micronaut.security.authentication.Authenticator;
+import io.micronaut.security.authentication.UsernamePasswordCredentials;
+import io.micronaut.security.event.LoginFailedEvent;
+import io.micronaut.security.event.LoginSuccessfulEvent;
+import io.micronaut.security.handlers.LoginHandler;
+import io.micronaut.security.rules.SecurityRule;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Optional;
+import java.util.HashSet;
+import java.util.Set;
+import java.net.URI;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Requires(env = Environments.LOCAL)
```

Comment:
`LOCAL` or `DEV`, I think...

Reply:
I found no reference to a DEV environment anywhere in the code.  Am I missing something?

Reply:
You are completely correct. It is specified on deployment though (see line 109):

https://github.com/objectcomputing/check-ins/blob/6d8cdb94c60b61f94f15456ebac2cd5c56887682/.github/workflows/gradle-build-development.yml#L109

You'll need to add it to `Environments` as `Environments.DEVELOP` or some such with a value of "dev"

Reply:
I used `Environment.DEVELOPMENT` here from micronaut.  Is that going to do what's required?